SOP/SOP-mdc.adoc

```diff
@@ -0,0 +1,81 @@
+ifdef::env-github[]
+:status:
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+:table-caption!:
+endif::[]
+
+:toc:
+:toc-placement!:
+
+= Mobile Developer Console - Standard Operating Procedures
+
+:toc:
+toc::[]
+
+== Overview
+
+The following guide outlines the steps required to manage and solve issues in the https://github.com/aerogear/mobile-developer-console[Mobile Developer Console] which is managed, installed and configured via the https://github.com/aerogear/mobile-developer-console-operator[Mobile Developer Console Operator].
+
+== Success Indicators
+
+All alerts should appear as green in the Prometheus Alert Monitoring.
+
+IMPORTANT: Before checking any of following steps see if the operator pod is running successfully. More info: https://github.com/aerogear/mobile-developer-console/blob/0.2.0/SOP/SOP-operator.adoc[MobileDeveloperConsoleOperatorDown]
+
+
+== Critical 
+
+=== MobileDeveloperConsoleDown or MobileDeveloperConsoleContainerDown
+
+. Switch to the Mobile Developer Console namespace by running `oc project <namespace>`. E.g `oc project mobile-developer-console`.
+. Follow the <<Check Operator>> steps.
+
+. Verify the MobileDeveloperConsole is deployed in the same namespace as the operator.
+.. Run `oc get MobileDeveloperConsole`
++
+Expected results:
```

Comment:
Missing steps for `MobileDeveloperConsolePodCPUHigh `  and `MobileDeveloperConsolePodMemoryHigh `
as to how to scale the pods.

Reply:
Yes, will add those.